capella_model_explorer/__main__.py

```diff
@@ -147,10 +139,10 @@ def run_local_dev() -> None:
         "*.py",
         "capella_model_explorer.app:app",
     ]
-    print(" ".join(uvicorn_cmd))
-    tailwind_proc = build_css(watch=True)
-    assert tailwind_proc is not None
-    time.sleep(1)  # avoid direct uvicorn reload when css file is written
+    logger.info(" ".join(uvicorn_cmd))
+    assert (tailwind_proc := build_css(watch=True))
+    time.sleep(1)
```

Comment:
Ruff claims: "Avoid assignment expressions in `assert` statements."

A deviation from the configuration in our cookiecutter to save one line of code?
See also my comment on that change in the file `pyproject.toml`.
I think this improvement is not great enough to leave an agreed stadard set of configuration options.